app/models/fae/static_page.rb

```diff
@@ -62,7 +62,12 @@ def self.define_association(name, type, locale_name = nil)
 
       send :has_one, name.to_sym, -> { where(attached_as: locale_name.to_s)}, as: poly_sym(type), class_name: type.to_s, dependent: :destroy
       send :accepts_nested_attributes_for, name, allow_destroy: true
-      send :define_method, :"#{name}_content", -> { send(name.to_sym).try(:content) }
+
+      if type == Fae::Checkbox
+        send :define_method, :"#{name}?", -> { send(name.to_sym).try(:content) }
+      else
+        send :define_method, :"#{name}_content", -> { send(name.to_sym).try(:content) }
+      end
```

Comment:
The conditional implies different functionality. What would you think of removing it to just what it is for, to make it DRYer, but also to remove complexity from the functionality.
```ruby
suffix = type == Fae::Checkbox ? '?' : '_content'
send :define_method, :"#{name}#{suffix}", -> { send(name.to_sym).try(:content) }
```
or
```ruby
method_name = if type == Fae::Checkbox
  :"#{name}?"
else
  :"#{name}_content"
end
send :define_method, method_name, -> { send(name.to_sym).try(:content) }
```